### DISCUSSION TITLE
[BUG] Problem with custom alerts

### DISCUSSION THREAD

**Discussion by fjpezuela** [NightscoutFoundation/xDrip#2282] [General]
In the latest versions of xDrip, the alarms cannot be customized. The explorer opens, but it does not allow you to select any type of file. I have tested it with several of the latest versions of xDrips and on two different phones (Android 10 and Android 12)

**Comment by Navid200** [comment]
This is not new.  
You can select and use a custom file as explained here for any of the xDrip alerts:
https://navid200.github.io/xDrip/docs/Custom-Alert-Sound.html

If you believe that something has changed in the recent releases, please identify the release and confirm that you have verified this by testing that release and the one before and have verified different behavior between those two releases.

**Reply by fjpezuela** [reply]
Yes, the problem is that it doesn't let you select the mp3 sounds. Sounds appear as "disabled" and won't let you press them. The funny thing is that if you already had a personalized ringtone from a version in which it worked, it still rings and the name appears, but you can't change it.

**Reply by Navid200** [reply]
I provided a link.  Did you look at it?

**Reply by Navid200** [reply]
Would you be so kind to read the link and try the suggested solution?

**Reply by fjpezuela** [reply]
Yes,

You mean they can only be selected if the files are copied to the "Notifications" folder? I will try it and tell you

**Reply by Navid200** [reply]
Yes, that is a work-around.
But, I need to show to the other developers that it works so that I can convince them to let me remove the malfunctioning "Custom Sound" option.

Thanks for testing.

**Reply by fjpezuela** [reply]
Sorry, I was testing it from the "custom" option. I have tried as indicated, copying the files in the "Notifications" folder and looking for the songs and if all the music files appear and let me select them. Thank you very much for your help.

**Reply by Navid200** [reply]
Sorry, does it work?

**Reply by fjpezuela** [reply]
Yes, work fine for me

**Reply by Navid200** [reply]
Thanks

**Reply by fjpezuela** [reply]
Thanks :)
